java/BUILD.bazel

```diff
@@ -39,6 +39,7 @@ define_java_module(
     name = "api",
     visibility = ["//visibility:public"],
     deps = [
+        "@maven//:com_google_guava_guava",
```

Comment:
Please remove this large dependency from api module. API dependencies should keep as simple as possible.

Reply:
Removed!